Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -510,6 +510,16 @@ theorem toNNReal_nat (n : ℕ) : (n : ℝ≥0∞).toNNReal = n := by
 theorem toReal_nat (n : ℕ) : (n : ℝ≥0∞).toReal = n := by
   rw [← ENNReal.ofReal_natCast n, ENNReal.toReal_ofReal (Nat.cast_nonneg _)]
 
+lemma _root_.Nat.cast_toNNReal (N : ℕ) :
+    (Nat.cast N : ℝ≥0∞).toNNReal = (Nat.cast N : ℝ).toNNReal := by
```

Comment:
I don't think `Nat.cast` needs to be written out; `(N : ℝ≥0∞).toNNReal = (N : ℝ).toNNReal` works